distribution/src/main/release/samples/jax_rs/spring_boot_scan/client/pom.xml

```diff
@@ -43,22 +52,23 @@
             <artifactId>org.qi4j.library.circuitbreaker</artifactId>
             <version>${cxf.zest.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Ok, this dependency was a bit of an annoyance.  Its defined in the samples parent, but it interferes with spring boot's use of logback.